metadata/src/main/java/org/apache/kafka/metadata/migration/KRaftMigrationDriver.java

```diff
@@ -403,7 +404,7 @@ public void run() throws Exception {
                 AtomicInteger count = new AtomicInteger(0);
                 zkMigrationClient.readAllMetadata(batch -> {
                     try {
-                        log.info("Migrating {} records from ZK: {}", batch.size(), batch);
+                        log.info("Migrating {} records from ZK", batch.size());
```

Comment:
do we want to log the batch itself when TRACE is active?

Reply:
I wondered about that. So far, we are avoiding logging raw records in most places, but it could be very useful for debugging during a migration. I'll add it back as TRACE.